src/lxc/lxccontainer.h

```diff
@@ -1017,6 +1017,22 @@ int list_active_containers(const char *lxcpath, char ***names, struct lxc_contai
  */
 int list_all_containers(const char *lxcpath, char ***names, struct lxc_container ***cret);
 
+struct lxc_log_conf {
+	const char *name;
+	char *file;
+	char *priority;
+	const char *prefix;
+	int quiet;
+	const char *lxcpath;
```

Comment:
Note that the order of the struct is important and should not be changed later as otherwise callers of the shared library might have the wrong offset if they were compiled against a different version.